proto/system_config.proto

```diff
@@ -183,6 +183,9 @@ message SwitchSetup {
   // Local port for an alternate OpenFlow controller
   int32 alt_port = 16;
 
+  // Gauge OpenFlow Port
+  int32 gauge_port = 15;
```

Comment:
this shouldn't expose the concept of "gauge" here, as that's an internal implementation detail.  How is this functionally different than lo_port? or alt_port?  Doesn't the switch just get programmed with two controllers? So would that be { lo_port, lo_port2 }?

Reply:
`lo_port_2` LTGM. Just wanted the user to be able to configure it in DAQ... Will update accordingly.

Reply:
Changed to `lo_port_2`. PTAL..